plugins/hls-hlint-plugin/src/Ide/Plugin/Hlint.hs

```diff
@@ -95,8 +101,8 @@ descriptor plId = (defaultPluginDescriptor plId)
       [ PluginCommand "applyOne" "Apply a single hint" applyOneCmd
       , PluginCommand "applyAll" "Apply all hints to the file" applyAllCmd
       ]
-    , pluginHandlers = mkPluginHandler STextDocumentCodeAction codeActionProvider
-    , pluginConfigDescriptor = defaultConfigDescriptor {configHasDiagnostics = True}
+  , pluginHandlers = mkPluginHandler STextDocumentCodeAction codeActionProvider
+  , pluginConfigDescriptor = defaultConfigDescriptor {configCustomConfig = mkCustomConfig properties}
```

Comment:
Why removing `configHasDiagnostics = True`?

Reply:
Ugh good catch

Reply:
Oops, sorry about that!